src/mkdocstrings_handlers/python/handler.py

```diff
@@ -235,11 +236,13 @@ def collect(self, identifier: str, config: Mapping[str, Any]) -> CollectorItem:
                 modules_collection=self._modules_collection,
                 lines_collection=self._lines_collection,
             )
-            try:
+            try:  # noqa: WPS229 we expect one type of exception, and want to fail on the first one
+                for pre_loaded_module in config.get("preload_external_modules", []):
```

Comment:
Lets use `final_config` for consistency, and because otherwise I think we wouldn't retrieve the global config. Since I suggest to use `None` rather than `False` in the default config, I use `or` to default to a empty list in case the config item is found and is `None`.

```suggestion
                for pre_loaded_module in final_config.get("preload_external_modules") or []:
```

Reply:
Done